.github/workflows/.windows.yml

```diff
@@ -338,6 +364,11 @@ jobs:
           If ("${{ matrix.runtime }}" -eq "containerd") {
             $runtimeArg="--containerd=\\.\pipe\containerd-containerd"
             echo "DOCKER_WINDOWS_CONTAINERD_RUNTIME=1" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+            # FIXME: This doesn't really put the daemon on containerd snapshotter mode, see https://github.com/moby/moby/pull/46402#issuecomment-1710232310
+            If ("${{ matrix.storage }}" -eq "snapshotter") {
+              echo "TEST_INTEGRATION_USE_SNAPSHOTTER=1" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+              echo "DOCKER_GRAPHDRIVER=windows" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+            }
```

Comment:
Fix is here: https://github.com/moby/moby/pull/46539/commits/c9c5594823a1c57a07734006ff41c1c7c5970274#diff-90e64dd59d8246305776c16fc6a5a006ef9458e5d0f7a6d64147f1c9ad97ddedL261-R283.

I removed `DOCKER_GRAPHDRIVER` as the Windows daemon doesn't honour it, and AFAIK doesn't plan to do so.

Reply:
Ok `TEST_INTEGRATION_USE_SNAPSHOTTER` is taken into account: https://github.com/moby/moby/actions/runs/6359507674/job/17272749812?pr=46572#step:21:72

```
Warning: 2023-09-30T02:28:08.8364201Z [Warning] Enabling containerd snapshotter through the $TEST_INTEGRATION_USE_SNAPSHOTTER environment variable. This should only be used for testing.
```

But looks like daemon fails to start: https://github.com/moby/moby/actions/runs/6359507674/job/17272749812?pr=46572#step:21:97

```
Error: 2023-09-30T02:28:08.9526619Z [Error] fatal: no host network support on this platform
```

Reply:
Ah I read your changes https://github.com/moby/moby/commit/c9c5594823a1c57a07734006ff41c1c7c5970274#diff-90e64dd59d8246305776c16fc6a5a006ef9458e5d0f7a6d64147f1c9ad97ddedL261-R283 wrong. Updated.

Reply:
Same fatal error: https://github.com/moby/moby/actions/runs/6359657942/job/17272998350?pr=46572#step:21:97